text/metrics/0001-naming-conventions.md

```diff
@@ -0,0 +1,147 @@
+# Metric naming conventions
+
+## Purpose
+
+Metric names and labels are the primary read-interface to metric data. The names and taxonomy need to be understandable and discoverable during routine exploration -- and this becomes critical during incidents.
+
+## Guidelines
+
+Namespace similar metrics together. Define top-level namespaces for common metric categories: for OS, like CPU and network; for app runtimes, like the JVM. This aids in discovery and adhoc comparison
+
+Provide consistent names for common labels. Provides discoverability and disambiguation similar to metric names.
+
+Avoid semantic ambiguity. Use namespaced metric names in cases where similar metrics have significantly different implementations across the holistics of metrics. For example, every garbage collected runtime has a slightly different strategies and measures. Using common metric names for GC, not namespaces by the runtime, could create dissimilar comparisons and confusion for end users. Measures of operating system memory are similar.
+
+## Conventions
+
+All metrics have a limited set of common labels:
+* `system.host`
+
+### Operation System
+
+#### CPU
+
+`system.cpu` is the core metric name.
+
+Standard usage labels include, non-exhaustively,
+* `cpu.idle`
+* `cpu.user`
+* `cpu.sys`
+* `cpu.real`
+* `cpu.iowait`
+* `cpu.nice`
+* `cpu.interrupt`
+* `cpu.softirq`
+* `cpu.steal`
```

Comment:
I am confused here, because a label has a key and a value and you are proposing only `values` (or I am just super confused)?

Reply:
Just the label keys, sorry if that was unclear.

Reply:
"cpu.sys" is a combination of key and value, not just a key.

Reply:
Resolving now -- specifics about metric names and labels will be covered in a future doc